### PR TITLE
Added configuration for the ask for default browser stage 1 experiment.

### PR DESCRIPTION
**Pull request by anikiki**
**Request**

* Android: https://app.asana.com/0/0/1205522646993015/f
_Note_: See also https://app.asana.com/0/0/1205697958017456/f

**Merge Schedule**

* Mon 16 Oct 2023 at 00:10AM ET time / 5:10AM London time
_Note_: The Atb values in the rules were updated and are accurate.

**Screenshots**

Screenshot_20231002-165230

### CODE REVIEW

**Review by joshliebe — APPROVED** [review]
lgtm 👍

File: live/android-config/android-config.json [hunk 1]
```diff
@@ -1,5 +1,130 @@
...
+            "en-CA",
+            "en-GB",
+            "en-AU"
+          ]
+        },
+        "defaultBrowser": {
+          "value": false
+        },
+        "atb": {
+          "value": "v399-3"
```
Comment (cmonfortep) [inline live/android-config/android-config.json:125]: Just noticed in the previous rules we are targeting -5 and -6, so I would assume this is -4? or whatever the atb should be, but sound like they are correlated.

File: live/android-config/android-config.json [hunk 2]
```diff
@@ -1,5 +1,130 @@
...
+      ]
+    }
+  ],
+  "rules": [
+    {
+      "id": 1,
+      "attributes": {
+        "locale": {
+          "value": [
+            "en-US",
```
Comment (cmonfortep) [inline live/android-config/android-config.json:77]: 👍  we are fine with these ones

File: live/android-config/android-config.json (hunk 1 above)
Comment (anikiki) [reply live/android-config/android-config.json:125]: The days are day0, day 1 and day 3. We don't target day 2.

**Pull request merged**